README.md

```diff
@@ -79,7 +133,8 @@ Please refer to [CONTRIBUTING.md][contributing].
 [add other sample code repo links]: # (Developer advocacy is open-sourcing a repo that has excellent sample code. The link will be added here.)
 
 [developer guide]: https://docs.snowflake.com/en/developer-guide/snowpark/python/index.html
-[api references]: https://docs.snowflake.com/en/developer-guide/snowpark/reference/python/index.html
+[Snowpark Python api references]: https://docs.snowflake.com/en/developer-guide/snowpark/reference/python/index.html
+[Snowpark pandas api references]: https://docs.snowflake.com/LIMITEDACCESS/snowpark-pandas#api-reference
```

Comment:
should we just use the future public link?

Reply:
That would be great, do we know what the link will be?

Reply:
Resolved offline, the link to the pandas api docs after PuPr will be: https://docs.snowflake.com/en/developer-guide/snowpark/reference/python/latest/modin/index.html

We should only merge this PR after the release is cut and docs are published.

Reply:
Update: The LIMITEDACCESS link will redirect so let's leave it and merge this PR.